tofu/tests/tests01_geom/tests03_core.py

```diff
@@ -856,22 +856,28 @@ def ffT(Pts, t=None, vect=None):
 
         ind = None#[0,10,20,30,40]
         minimize = ["memory", "calls", "hybrid"]
-        for mmz in minimize:
-            for typ in self.dobj.keys():
-                for c in self.dobj[typ].keys():
-                    obj = self.dobj[typ][c]
-                    ff = ffT if obj.config.Id.Type=='Tor' else ffL
-                    t = np.arange(0,10,10)
-                    connect = (hasattr(plt.get_current_fig_manager(),'toolbar')
-                               and getattr(plt.get_current_fig_manager(),'toolbar')
-                               is not None)
-                    out = obj.calc_signal(ff, t=t, ani=True, fkwdargs={},
-                                          res=0.01, DL=None, resMode='abs',
-                                          method='simps', minimize=mmz, ind=ind,
-                                          plot=False, out=np.ndarray,
-                                          fs=(12,6), connect=connect)
-                    sig, units = out
-                    assert not np.all(np.isnan(sig)), str(ii)
+        for aa in [True, False]:
+            for rm in ["abs", "rel"]:
+                for dm in ["simps", "romb", "sum"]:
+                    for mmz in minimize:
```

Comment:
These loops make the test much slower but it's the most complete test we can have... toss or keep?

Reply:
Keep, and if want faster tests at some point we can temporarily comment them